### PR TITLE
Add WordReference plugin

### PR DESCRIPTION
**Pull request by LeoDupont**
Hi

This is a simple WordReference translation plugin written with JavaScript.
Here is the repo: https://github.com/LeoDupont/Flow.Launcher.Plugin.WordReference

Suggestions welcome!

### CODE REVIEW

**Comment by jjw24** [comment]
Checked out the code lgtm, will test run it when I'm in front of computer.

Thanks for adding installing node.js instruction in the readme, when we get a couple more js plugins we can automate the install process.

File: plugins.json [hunk 1]
```diff
@@ -796,5 +796,19 @@
         "UrlSourceCode": "https://github.com/Garulf/flow-raindrop/tree/main",
         "IcoPath": "https://cdn.jsdelivr.net/gh/Garulf/flow-raindrop@main/icon.png",
         "Tested": true
+    },
+    {
+        "ID": "90170ddfd94a4c188bc5ff2e3690d0ae",
+        "ActionKeyword": "*",
```
Comment (jjw24) [inline plugins.json:802]:
```suggestion
```

File: plugins.json [hunk 2]
```diff
@@ -796,5 +796,19 @@
...
+        "Name": "WordReference",
+        "Description": "WordReference translations (es, en, fr, it)",
+        "Author": "LeoDupont",
+        "Version": "1.0.0",
+        "Language": "executable",
+        "Website": "https://github.com/LeoDupont/Flow.Launcher.Plugin.WordReference",
+        "UrlDownload": "https://github.com/LeoDupont/Flow.Launcher.Plugin.WordReference/releases/download/v1.0.0/Flow.Launcher.Plugin.WordReference.zip",
+        "UrlSourceCode": "https://github.com/LeoDupont/Flow.Launcher.Plugin.WordReference/tree/main",
+        "ExecuteFileName": "start.bat",
```
Comment (jjw24) [inline plugins.json:811]:
```suggestion
```

**Review by jjw24 — APPROVED** [review]
Just removed two obsolete fields not required for submission.

👍

**Comment by LeoDupont** [comment]
Thank you!

**Pull request merged**